contracts/strategies/qv-hackathon/HackathonQVStrategy.sol

```diff
@@ -0,0 +1,269 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity 0.8.19;
+
+import {IAllo} from "../../core/IAllo.sol";
+import {IRegistry} from "../../core/IRegistry.sol";
+import {QVSimpleStrategy} from "../qv-simple/QVSimpleStrategy.sol";
+import {Metadata} from "../../core/libraries/Metadata.sol";
+
+import {EnumerableSet} from "@openzeppelin/utils/structs/EnumerableSet.sol";
+
+// Note: EAS Contracts
+import {
+    AttestationRequest,
+    AttestationRequestData,
+    IEAS,
+    Attestation,
+    MultiAttestationRequest,
+    MultiRevocationRequest
+} from "@ethereum-attestation-service/IEAS.sol";
+import {ISchemaRegistry, ISchemaResolver, SchemaRecord} from "@ethereum-attestation-service/ISchemaRegistry.sol";
+
+contract HackathonQVStrategy is QVSimpleStrategy {
+    using EnumerableSet for EnumerableSet.AddressSet;
+
+    error ALREADY_ADDED();
+
+    // The instance of the EAS and SchemaRegistry contracts.
+    IEAS public eas;
+    ISchemaRegistry public schemaRegistry;
+
+    bytes32 constant EMPTY_UID = 0;
+
+    // Who can call submitAttestation
+    mapping(address => bool) public attestationSigners;
+
+    EnumerableSet.AddressSet private hackers;
+
+    event EASAddressUpdated(address indexed easAddress);
+    event VerifierAdded(address indexed verifier);
+    event VerifierRemoved(address indexed verifier);
+
+    constructor(address _allo, string memory _name) QVSimpleStrategy(_allo, _name) {}
+
+    /// @dev Initializes the strategy
+    /// @param _poolId The pool ID for this strategy
+    /// @param _data The data to initialize the strategy with
+    function initialize(uint256 _poolId, bytes memory _data) public override {
+        // decode the _data -> the second data has the timestamps and QVSimpleStrategy data
+        (
+            address easContractAddress,
+            address schemaRegistryAddress,
+            string memory schema,
+            bool revocable,
+            bytes memory data
+        ) = abi.decode(_data, (address, address, string, bool, bytes));
+
```

Comment:
This part is missing:

- **Allocator Eligibility**
    - Wallets are only eligible if they hold a specific NFT.
        - Pool managers must have the ability to configure which NFT is required in order to vote.

Reply:
This part is missing:


### Payout calculation logic
The `payout` for each recipient is determined by their rank in total number of received votes (NOT VOICE CREDITS). 
- the pool admin can configure which rank positions receive funds and what percentage of the pool each place gets.
    - For example, the pool admin can say that first place gets 50%, second place gets 30%, and so on.
    
----
How do we track the places without heavy on-chain calculations? any ideas @thelostone-mc @codenamejason